pkg/go/go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/openfga/language/pkg/go
 
-go 1.21.5
+go 1.20
```

Comment:
What's causing is to downgrade?

Reply:
A third party faced an issue with us using go1.21, we're gonna do a latest - 1 approach, so support the latest minor + the one before it. So while current is go1.21, we'll support go1.20+

See here: https://github.com/lxc/incus/issues/315#issuecomment-1863381991

Reply:
Understood:) thanks for the reference. Should we then publicly follow the same release policy as Go itself and state it publicly within all our customer facing SDKs/Tools https://go.dev/doc/devel/release#policy?

Reply:
I dunno if we want to publicly commit to it yet versus a "best effort". At least for now